drivers/media/platform/raspberrypi/rp1_cfe/cfe_fmts.h

```diff
@@ -197,24 +197,28 @@ static const struct cfe_fmt formats[] = {
 		.code = MEDIA_BUS_FMT_SBGGR16_1X16,
 		.depth = 16,
 		.flags = CFE_FORMAT_FLAG_FE_OUT,
+		.remap = { 0, V4L2_PIX_FMT_PISP_COMP1_BGGR },
```

Comment:
For the remap, instead of 0, should we use V4L2_PIX_FMT_*?  Not sure it matters.